WMFPageHistorySection.h

```diff
@@ -0,0 +1,17 @@
+//
+//  WMFPageHistorySection.h
+//  Wikipedia
+//
+//  Created by Nick DiStefano on 4/3/16.
+//  Copyright © 2016 Wikimedia Foundation. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+@class WMFPageHistoryRevision;
+
+@interface WMFPageHistorySection : NSObject
+
+@property (nonatomic, copy, readwrite) NSString* _Nullable sectionTitle;
```

Comment:
@coreyfloyd sectionTitle can be immutable. The way the items are constructed now the object needs to allow a new item to be added. The parsing could be reconsidered to allow this to be immutable, but that might be more trouble than it's worth. I'll take another look at that later on. 

not 100% related, but I think there is room for improvement in the way dictionaries are constructed in swift in general. I like python's init with array of tuples. https://github.com/apple/swift-evolution/pull/125 is a proposal along those lines. might be worth rolling a version of that if there is a bunch more dictionary manipulation to be done